rulesets/MetabolizingProperty.cpp

```diff
@@ -0,0 +1,407 @@
+// Cyphesis Online RPG Server and AI Engine
+// Copyright (C) 2013 Alistair Riddoch
```

Comment:
You should probably add yourself as an author here too.

Now, for most files that I alter I don't do this (Git contains the record anyway), but if it's a new file you should have your name there too.